packages/shared/lib/contexts/onboarding/actions/claimShimmerRewards.ts

```diff
@@ -60,7 +60,7 @@ async function claimShimmerRewardsForShimmerClaimingAccounts(
 async function claimShimmerRewardsForShimmerClaimingAccount(
     shimmerClaimingAccount: IShimmerClaimingAccount
 ): Promise<void> {
-    const recipientAddress = shimmerClaimingAccount?.twinAccount?.meta?.publicAddresses[0]?.address
+    const recipientAddress = (await shimmerClaimingAccount?.twinAccount?.addresses())[0].address
```

Comment:
do we need the following?
(await shimmerClaimingAccount?.twinAccount?.addresses())?.[0].address

Reply:
Good catch, we need the first address which is external. I added a utils function to account that fetches this, since we use it in two places.